### PR TITLE
[ci skip] add note for individual stub creation

### LINKED ISSUE #20694: A Guide to Testing Rails Applications: generate unit test if controller and/or model exist
I did not find this information in documentation but often facing projects without tests I think it is very useful information:

rails g test_unit:scaffold user
It will generate controller test with filled all usual actions.
      create  test/controllers/users_controller_test.rb

rails g test_unit:model user
      create  test/models/user_test.rb
      create  test/fixtures/users.yml

### PR DESCRIPTION
**Pull request by aditya-kapoor**
Closes #20694

### CODE REVIEW

File: guides/source/testing.md [hunk 1]
```diff
@@ -178,6 +178,16 @@ create  test/fixtures/articles.yml
 ...
 ```
 
+You can also generate the test stub for an existing model or new model using following command:
```
Comment (jvanbaarsen) [inline guides/source/testing.md]: I think it should be "using *the* following command" right?

File: guides/source/testing.md [hunk 2]
```diff
@@ -509,6 +519,13 @@ You should test for things such as:
 
 Now that we have used Rails scaffold generator for our `Article` resource, it has already created the controller code and tests. You can take look at the file `articles_controller_test.rb` in the `test/controllers` directory.
 
+The following command will generate controller test with default seven actions filled up.
```
Comment (jvanbaarsen) [inline guides/source/testing.md]: I think it should be "will generate *a* controller test" right?

**Comment by jvanbaarsen** [comment]
Looks like an useful addition! /cc @senny

**Comment by aditya-kapoor** [comment]
@jvanbaarsen PR updated...

File: guides/source/testing.md [hunk 3]
```diff
@@ -178,6 +178,16 @@ create  test/fixtures/articles.yml
 ...
 ```
 
+You can also generate the test stub for an existing model or new model using the following command:
```
Comment (robin850) [inline guides/source/testing.md]: You can certainly just say "for a model using the ...".

File: guides/source/testing.md [hunk 4]
```diff
@@ -178,6 +178,16 @@ create  test/fixtures/articles.yml
...
+
+```bash
+$ bin/rails generate test_unit:model article title:string body:text
+...
```
Comment (robin850) [inline guides/source/testing.md]: :scissors:

File: guides/source/testing.md [hunk 5]
```diff
@@ -178,6 +178,16 @@ create  test/fixtures/articles.yml
...
+create  test/models/article_test.rb
+create  test/fixtures/articles.yml
+...
```
Comment (robin850) [inline guides/source/testing.md]: :scissors:

File: guides/source/testing.md [hunk 6]
```diff
@@ -509,6 +519,13 @@ You should test for things such as:
 
 Now that we have used Rails scaffold generator for our `Article` resource, it has already created the controller code and tests. You can take look at the file `articles_controller_test.rb` in the `test/controllers` directory.
 
+The following command will generate a controller test with default seven actions filled up.
```
Comment (robin850) [inline guides/source/testing.md]: What do you think about:

> The following command will generate a controller test case with a filled up test for each of the seven default actions.

? Also make sure to wrap lines at 80 chars.

**Comment by aditya-kapoor** [comment]
@robin850 PR Updated... :smile:

File: guides/source/testing.md [hunk 7]
```diff
@@ -178,6 +178,15 @@ create  test/fixtures/articles.yml
 ...
 ```
 
+You can also generate the test stub for for a model using
+the using the following command:
```
Comment (robin850) [inline guides/source/testing.md]: You can remove the "the using".

File: guides/source/testing.md [hunk 8]
```diff
@@ -507,7 +516,17 @@ You should test for things such as:
...
-Now that we have used Rails scaffold generator for our `Article` resource, it has already created the controller code and tests. You can take look at the file `articles_controller_test.rb` in the `test/controllers` directory.
+Now that we have used Rails scaffold generator for our `Article` resource,
+it has already created the controller code and tests. You can take
+look at the file `articles_controller_test.rb` in the `test/controllers` directory.
+
+The following command will generate a controller test case with a filled up
+test for each of the seven default actions.
+
+```bash
+$ rails g test_unit:scaffold article
```
Comment (robin850) [inline guides/source/testing.md]: Could you add the "bin/" prefix here and use "generate" instead of "g" to be consistent please ?

**Comment by aditya-kapoor** [comment]
@robin850 PR Updated..

File: guides/source/testing.md [hunk 9]
```diff
@@ -178,6 +178,14 @@ create  test/fixtures/articles.yml
 ...
 ```
 
+You can also generate the test stub for for a model using the following command:
```
Comment (robin850) [inline guides/source/testing.md]: Okay, one last thing, there's a double "for" here.

Comment (aditya-kapoor) [inline guides/source/testing.md]: done... :+1:

**Comment by robin850** [comment]
Thanks!

**Pull request merged**